templates/resources/index.html

```diff
@@ -0,0 +1,15 @@
+{% extends "templates/one-column.html" %}
+
+{% block meta_description %}When an Ubuntu release reaches its end of life it receives no further maintenance updates, including critical security upgrades. It is highly recommended that you upgrade to a recent version of Ubuntu at this point.{% endblock %}
```

Comment:
Do we need this at this stage, I'm worried it might get missed down the line?

Reply:
Good spot, updated.